packages/nextra-theme-docs/src/components/sidebar.tsx

```diff
@@ -84,9 +84,9 @@ function FolderImpl({ item, anchors }: FolderProps) {
       <ArrowRightIcon
         height="1em"
         className={cn(
-          'ml-2 h-[18px] min-w-[18px] rounded-sm p-[2px] hover:bg-gray-800/5 dark:hover:bg-gray-100/5',
-          '[&>path]:origin-center [&>path]:transition-transform',
-          open && '[&>path]:rotate-90'
+          'h-[18px] min-w-[18px] rounded-sm p-0.5 hover:bg-gray-800/5 dark:hover:bg-gray-100/5',
+          '[&>path]:origin-center [&>path]:transition-transform rtl:[&>path]:-rotate-180',
+          open && 'ltr:[&>path]:rotate-90 rtl:[&>path]:-rotate-90'
```

Comment:
fixes, icon should be inverted -90deg
<img width="285" alt="image" src="https://user-images.githubusercontent.com/7361780/184453008-7d09cef9-1bbb-42c1-af33-d1df668beb69.png">